frontend/views/search/_results.html.erb

```diff
@@ -5,20 +5,33 @@
 <%
   # Start of modifications
   # See https://github.com/archivesspace/archivesspace/blob/master/frontend/app/helpers/search_helper.rb
+  # Customize columns in search results if accessions might be included
+
   model = "accession"
-  add_pref_columns(model)
-  # Add content_description column at end
-  prop = 'content_description'
-  enum_locale_key = "#{model}_#{prop}"
-  add_column(
-    I18n.t("#{model}.#{prop}"),
-    { locale_key: enum_locale_key, model: model, sortable: false },
-    proc { |record|
-      v = record[prop] || ASUtils.json_parse(record['json'])[prop]
-      I18n.t("enumerations.#{enum_locale_key}.#{v}", :default => v.to_s) unless v.nil?
-    }
-  )
-  add_actions_column if !params[:linker] || params[:linker] === 'false'
+  if !@search_data.nil? & [model, "multi"].include?(@search_data.get_type)
```

Comment:
[`@search_data.get_type`](https://github.com/archivesspace/archivesspace/blob/305b9a3a11579f28472e0a4948803c2a7a8e959d/frontend/app/models/search_result_data.rb#L215-L234) returns `multi` if there are multiple resource types returned by the search, or the type name if there's just one. Checking for `accession` and `mutli` here means that Clements will get the custom column they want whenever accessions MIGHT be in the results; otherwise, previous column behavior remains, including a useful "Resource Type" column when there are multiple types. 

This should fix and improve upon the previous behavior I introduced, where search results ALWAYS had accessions columns and the "Content Description".